### PR TITLE
chore(plugin/model): prepare examples for 7.18.0

### PR DESCRIPTION
**Pull request by mboskamp**
Related to CAM-14550

### CODE REVIEW

File: bpmn-model-api/generate-jsf-form/pom.xml [hunk 1]
```diff
@@ -15,6 +15,8 @@
     <camunda.version>7.18.0</camunda.version>
     <arquillian.version>1.5.0.Final</arquillian.version>
     <wildfly.container.adapter.version>2.2.0.Final</wildfly.container.adapter.version>
+	<version.junit>4.13.1</version.junit>
+	<version.slf4j>1.7.26</version.slf4j>
```
Comment (koevskinikola) [inline bpmn-model-api/generate-jsf-form/pom.xml:19]: @mboskamp should we fix the formatting here?

File: bpmn-model-api/generate-process-fluent-api/pom.xml [hunk 2]
```diff
@@ -8,6 +8,10 @@
 
   <properties>
     <camunda.version>7.18.0</camunda.version>
+	<version.junit>4.13.1</version.junit>
+	<version.h2>2.1.214</version.h2>
+	<version.slf4j>1.7.26</version.slf4j>
+	<version.servlet-api>4.0.1</version.servlet-api>
```
Comment (koevskinikola) [inline bpmn-model-api/generate-process-fluent-api/pom.xml:14]: Same here.

File: cmmn-model-api/typed-custom-elements/pom.xml [hunk 3]
```diff
@@ -11,6 +11,10 @@
 
   <properties>
     <version.camunda>7.18.0</version.camunda>
+	<version.junit>4.13.1</version.junit>
+	<version.assertj>2.9.1</version.assertj>
+	<version.h2>2.1.214</version.h2>
+	<version.slf4j>1.7.26</version.slf4j>
```
Comment (koevskinikola) [inline cmmn-model-api/typed-custom-elements/pom.xml:17]: Maybe we can adjust the formatting on these rows.

**Review by koevskinikola — APPROVED** [review]
It would be good if we can adjust the formatting of the new `pom.xml` properties. Otherwise, the changes look good.

File: bpmn-model-api/generate-jsf-form/pom.xml (hunk 1 above)
Comment (koevskinikola) [inline bpmn-model-api/generate-jsf-form/pom.xml:19]:
```suggestion
    <wildfly.container.adapter.version>2.2.0.Final</wildfly.container.adapter.version>
    <version.junit>4.13.1</version.junit>
    <version.slf4j>1.7.26</version.slf4j>
```

**Comment by mboskamp** [comment]
@koevskinikola, as discussed I changed the tabs to spaces. Can you check again if everything is aligned now?

**Pull request merged**